src/cli/cli.cpp

```diff
@@ -4544,6 +4547,24 @@ otError Interpreter::ProcessMacSend(Arg aArgs[])
 }
 #endif
 
+#if OPENTHREAD_CONFIG_RADIO_LINK_TREL_ENABLE
+otError Interpreter::ProcessTrel(Arg aArgs[])
+{
+    otError error = OT_ERROR_INVALID_COMMAND;
+
+    if (aArgs[0] == "enable")
```

Comment:
may show the state of trel when no arg?

Reply:
not possible because there is no platform API to get the `TestMode` state.

Reply:
Not sure if there is scenario that tester need to check trel state, if there is, may consider adding one API?

Reply:
I think it's a good idea to allow checking TREL state, but we can do it later if it turn out to be necessary.

For now, the TREL state is determinate: 
- If we start `otbr-agent` with a TREL URL, then TREL is `enabled` by default, and can be disabled and enabled later.
- If we start `otbr-agnet` without a TREL URL, then TREL is `disabled` by default and can not be enabled. 

So, the TREL state is actually known beforehand without a checking.

Reply:
Fine to leave it to future when required.